src/patternfly/components/Page/page.scss

```diff
@@ -247,7 +254,6 @@ $pf-page-v6--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg
   &.pf-m-limit-width {
     display: flex;
     flex-direction: column;
```

Comment:
Since body is required now, wdyt about moving these styles to this block so all of the main-[whatever] sections (anything that has `__main-body` as children) have this style?